controller/model/authenticator_mod_ext_jwt.go

```diff
@@ -67,31 +70,39 @@ func (a *AuthModuleExtJwt) pubKeyLookup(token *jwt.Token) (interface{}, error) {
 	kidVal, ok := token.Header["kid"]
 
 	if !ok {
+		pfxlog.Logger().Error("missing kid")
```

Comment:
saddest log message ever :cry:

Reply:
Haha, reading that out of context only made me second guess why that message exists.